source/en/latest/developer/flatpak-repo.markdown

```diff
@@ -0,0 +1,58 @@
+---
+Type:            article
+Title:           Maintaining a HandBrake flatpak repository
+Project:         HandBrake
+Project_URL:     https://handbrake.fr/
+Project_Version: Latest
+Language:        English
+Language_Code:   en
+Text_Encoding:   utf-8
+Authors:         John (j45)
+Copyright:       2016 HandBrake Team
+License:         Creative Commons Attribution-ShareAlike 4.0 International
+License_Abbr:    CC BY-SA 4.0
+License_URL:     https://handbrake.fr/docs/license.html
+---
+
+Maintaining a HandBrake flatpak repository
+==========================================
+
+## Creating a new empty flatpak repository
+This would be done once to set up the repo users update their HandBrake
+flatpaks from. This repo gets served via http to users.
+
+Initialize a new repository  
+`ostree init --mode=archive-z2 --repo=<repo-dir>`
```

Comment:
Umm, this *is* a special case.  It's a code block preceded by a description of that code block.  The 2 things are associated, but must be on separate lines.  It makes sense for the code block to be physically closer to the line that describes it than to the things below for readability.